netbox/netbox/filtersets.py

```diff
@@ -325,3 +327,60 @@ def search(self, queryset, name, value):
             models.Q(slug__icontains=value) |
             models.Q(description__icontains=value)
         )
+
+
+class ScopedFilterSet(BaseFilterSet):
```

Comment:
This should probably live in the `dcim` app. This module is intended for filtersets pertaining to the base model classes.